include/global.h

```diff
@@ -169,7 +169,11 @@ struct UCoords32
 
 struct SaveBlock3
 {
-};
+#if USE_DEXNAV_SEARCH_LEVELS == TRUE
```

Comment:
config/dexnav.h needs to be included in constants/global for this file to read this.